Model/IO/Export/Nastran/BulkData/ShortBulkData.h

```diff
@@ -0,0 +1,83 @@
+//*************************
+//Created by Dominik Kopaczka
+//*************************
+
+#ifndef CPP_SHORTBULKDATA_H
+#define CPP_SHORTBULKDATA_H
+
+#include"BulkData.h"
+
+namespace MeshIS
+{
+	namespace Model
+	{
+		namespace Export
+		{
+			namespace Nastran
+			{
+				namespace BData
+				{
+					class ShortBulkData :public BulkData
+					{
+					#pragma region Constructors
```

Comment:
#pragma region is MSVC specyfic and not-portable. AFAIK it is not a part of standard.